mod/invitefriends/actions/invitefriends/invite.php

```diff
@@ -90,10 +87,10 @@
 	if (count($already_members) > 0) {
 		register_error(elgg_echo('invitefriends:already_members', array(implode(', ', $already_members))));
```

Comment:
hmmm leaving this as is for now... don't know what we would do with the error that is passed to the Response when it is an array... you could do a follow up PR if you have an idea how to approach this